src/CoolBoard.cpp

```diff
@@ -532,55 +532,26 @@ void CoolBoard::onLineMode()
 	Serial.println( F("Re-checking RTC..."));
 	rtc.update();
 
-	//read user data if user is active
-	if(userActive)
-	{
-		coolBoardLed.fadeIn(245,237,27,0.5);//shade of yellow
-	
-	#if DEBUG == 1
-
-		Serial.println( F("User is Active") );
-		Serial.println( F("Collecting User's data ( mac,username,timeStamp )") );
-		Serial.println();
-	
-	#endif	
-		coolBoardLed.blink(245,237,27,0.5);//shade of yellow	
-
-		//reading user data
-		data=this->userData();//{"":"","":"","",""}
+	//reading board data
+	data=this->boardData();//{"":"","":"","",""}
 
-		//formatting json 
-		data.setCharAt( data.lastIndexOf('}') , ',');//{"":"","":"","","",
-				
-		//read sensors data
-	#if DEBUG == 1
+	//formatting json 
+	data.setCharAt( data.lastIndexOf('}') , ',');//{"":"","":"","","",
+			
+	//read sensors data
+#if DEBUG == 1
 
-		Serial.println( F("Collecting sensors data ") );
-		Serial.println();
-	
-	#endif
+	Serial.println( F("Collecting sensors data ") );
+	Serial.println();
 
-		data+=this->readSensors();//{"":"","":"","","",{.......}		
+#endif
 
-		//formatting json correctly
-		data.remove(data.lastIndexOf('{'), 1);//{"":"","":"","","",.......}
-		
-		coolBoardLed.fadeOut(245,237,27,0.5);//shade of yellow
-				
-	}	
-	else
-	{
-		//read sensors data
-	#if DEBUG == 1
+	data+=this->readSensors();//{"":"","":"","","",{.......}		
```

Comment:
What's the use for the commented JSON behind ?

Reply:
to keep track of the hand-constructed json string